src/utils/getTodoFilter.ts

```diff
@@ -0,0 +1,17 @@
+import { Filter } from '../types/Filter';
+import { Todo } from '../types/Todo';
+
+export const getTodoFilter = (todos: Todo[], filter: Filter) => {
+  return todos.filter(todo => {
+    switch (filter) {
+      case Filter.Active:
+        return !todo.completed;
+
+      case Filter.Completed:
+        return todo.completed;
+
+      default:
+        return true;
+    }
+  });
```

Comment:
it is better to move filter inside switch case not vice versa to avoid extra iteration in defaul case